.Rbuildignore

```diff
@@ -14,3 +14,4 @@ notes.md
 ^revdep$
 vignettes/
 inst/ignore/
+docker-compose.yml
```

Comment:
Do the GitHub Actions workflows need to be updated?

Reply:
Eventually yes. 

From what I was able to understand I think the current Github Actions workflow is broken. So my plan would be first to make sure all tests are passing locally, and then update the GA workflows (which for me are more complex) in a different PR.

However, I wasn't able to test against multiple CKAN versions and still reuse the docker compose setup for Github Actions.